src/lib/grid-list/grid-list.md

```diff
@@ -41,3 +41,9 @@ however, more rows will simply be added for it the tile to fill.
 
 A header and footer can be added to an `md-grid-tile` using the `md-grid-tile-header` and
 `md-grid-tile-footer` elements respectively.
+
+### Accessibility
+Non-interactive grid list for layout purpose only should add `role=“list”` to the grid list.
```

Comment:
Nit: "Non-interactive grid lists for layout purposes only should have `role="list"`."